dumper.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# webhook2kafka
+# Copyright(C) 2019 Christoph Görn
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This will send all the webhook payloads to a Kafka topic."""
+
+import os
+import sys
+import ssl
+import logging
+import json
+
+
+import faust
+
+
+from thoth.common import init_logging
+from thoth.messaging import PackageRelease, THOTH_PACKAGE_RELEASES_TOPIC_NAME
+
+
+__version__ = "0.1.0-dev"
+
+_DEBUG = os.getenv("DEBUG", False)
+
+
+init_logging()
+_LOGGER = logging.getLogger("dumper")
```

Comment:
With faust apps you have to use app.logger, like this 
```python
app.log.debug("Failed to publish with the following error message: %r", identifier.msg)
```

Reply:
Is there any faust feature enforcing this? Generally this depends on the logger you want to use to log your messages.

The logger for all Thoth components/libraries live in "thoth" namespace, this enables as to transparently configure logging facilities across the whole application (see thoth-common readme).

The logger should be named:

```
_LOGGER = logging.getLogger("thoth.dumper")
```